CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ set(SOURCE_FILES
         src/shaka_scheme/system/base/String.cpp
         src/shaka_scheme/system/base/Symbol.cpp
         src/shaka_scheme/system/base/DataPair.cpp
-        src/shaka_scheme/system/exceptions/MissingImplementationException.hpp)
+        src/shaka_scheme/system/exceptions/MissingImplementationException.hpp src/shaka_scheme/system/base/Environment.cpp src/shaka_scheme/system/base/Environment.hpp)
```

Comment:
Remove `src/shaka_scheme/system/base/Environment.hpp`. Headers should not be in the source files.